app/server/__init__.py

```diff
@@ -39,6 +40,9 @@
                     backend=config.REDIS_URL,
                     task_serializer='json')
 
+i18n.load_path.append(os.path.abspath(os.path.join(os.getcwd(), "app/server/locale")))
```

Comment:
@enjeyw this was what was causing test failures. had to change to abs path instead of just the string for docker. so locally when i invoke tests, i'm doing it from root folder. will this cause issues for when we run the server, which happens from the /app folder?